### ISSUE TITLE
"chunk" has deleted my cable

### ISSUE THREAD

**Issue by 648643579** [GTNewHorizons/NewHorizons#3673]
Date: 26.09.2018
Server: Delta
Time: 17:30-19:42
https://i.imgur.com/WxNh01f.png

my base has 4 chunk loaders "Personal Anchor" (I use coins that are still active for two days) today they have not expire or change.
https://i.imgur.com/XBPFRLv.png
the red arrow is labeled chunk where the problem has become.

PS:
the cable was plugged in correctly, (there was no change in machines or cables)

**Comment by richardhendricks** [comment]
You are 100% sure you did not overvolt or overcurrent the cable?  How many generators and machines do you have attached to the wire?

In general, it's not wise to cross chunks with GT cables due to how the server handles slowing down ticks in unimportant chunks.  It's better to send liquid power and generate locally, or place transformers/battery buffers on each chunk transition.   I am not sure how it responds to chunkloading though.  

From what I understand, the issue is caused by GT power being a push system, and the server not necessarily giving every chunk an equal amount of ticks.  IE, chunk A has a generator, the generator sees a machine in chunk B that needs EU, so it sends a packet, but the packet gets transferred to the first cable in chunk B and sits there, since chunk B doesn't tick as fast as chunk A. And the generator sends another packet since the machine in B hasn't filled up yet,, etc. so eventually enough packets end up in the first cable in chunk B that it catches fire once it ticks and it sends packets downstream.

**Comment by 648643579** [comment]
yes I'm 100% (∞) sure. that I didn't do anything to break the cable.
https://i.imgur.com/991zeVR.png
https://i.imgur.com/eWL7fqw.png
https://i.imgur.com/FzKsmms.png
https://i.imgur.com/6udtg0j.png

**Comment by boubou19** [comment]
it's the good old chunk desyncing bug that burn cables if the chunks are desynced (ie chunks unloaded/loaded at a different speeds) It happens only on thermos servers so best you can do is always chunkload the area sadly

**Comment by noobyaran9** [comment]
![2018-09-22_11 06 46](https://user-images.githubusercontent.com/13598309/46255602-84cc7180-c4d1-11e8-8af3-185085490993.png)

Welp I lost a chunk of data.

**Comment by 648643579** [comment]
This problem reappeared:
Date: 11.11.2018
Server: Delta
Time: 16:05

https://vgy.me/RgN0jH.png
https://vgy.me/xetFAr.png

31 cables were lost.
https://vgy.me/oaubTR.png

**Comment by boubou19** [comment]
it's because the chunk got loaded at a different speed than its neighboors. a ductape fix is to always chunkload the area

**Comment by 648643579** [comment]
The chunks where the problem happened are read by one "Personal Anchor"
https://vgy.me/OzUK9R.png

do you mean that "Personal Anchor" load everyone with nine chunks differently? Rly?

**Comment by MakinToge** [comment]
not all 9 chunks are loaded at the exact same time. cable between chunk border is just a wish for cable loss... always go through a battery buffer or a transformer like this :
====|x=>
= is cable       | is chunk         x is buffer          > is eu flow direction

**Comment by 648643579** [comment]
@MakinToge 
do you mean this way?
https://vgy.me/P4WPO0.png

**Comment by MakinToge** [comment]
The root cause of the cable burning, is because the other chunk cable don't have the same tick, it may receive twice the amps from previous chunk and thus, burn. Doubling the amps capacity of the cable will remove most problems but it's still not 100% safe.
GT machines won't explode if there's more amps that what they need in the cable powering them so if the buffer receive twice or thrice as much amps from previous chunk, it won't have any problems and just output what their batteries can.
TL;DR yes.

**Comment by richardhendricks** [comment]
Mostly right.  The source generator would put the amps into the first block in the destination chunk.  So with a transformer/battery buffer as the first item on that side, it would collect the EU until it fills up.  If it was a cable, the cable gets EU dumped into it over and over since it never gets to the destination target further down.

Having a cable 2x the size of the source generator helps, but putting a transformer/battery buffer would prevent it from happening at all.
